code/modules/shuttle/super_cruise/orbital_poi_generator/objective_types/vip_extraction.dm

```diff
@@ -24,6 +28,9 @@
 		//Recovered and alive
 		if(is_station_level(mob_to_recover.z) && mob_to_recover.stat == CONSCIOUS)
 			complete_objective()
+		//Dead and no ckey
```

Comment:
```suggestion
		//Dead and has a ckey
```

Reply:
Not what we're checking for in this instance